viz-lib/src/visualizations/chart/Editor/SeriesSettings.jsx

```diff
@@ -64,6 +69,7 @@ function getTableColumns(options, updateSeriesOption, debouncedUpdateSeriesOptio
           data-test={`Chart.Series.${item.key}.Type`}
           dropdownMatchSelectWidth={false}
           value={item.type}
+          filterTypes={restrictedChartTypes}
```

Comment:
If it's the only place where `restrictedChartTypes` is used - just put the array here